tests/regression/54-unroll_arrays/01-simple_array.c

```diff
@@ -0,0 +1,40 @@
+// PARAM: --set solver td3 --enable ana.int.interval --set exp.arrays-domain unroll --set exp.array-unrolling-factor 5
+int global;
```

Comment:
Here and in the other added test:
```suggestion
#include <assert.h>
int global;
```

Reply:
Fixed on 1ad669514c033c4d1791608a3d8a64d06d1ba6d0